CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+[v1.1.7] - 2019-08-18
```

Comment:
The following corresponding line needs to be added to end of file (around linke 420) to make [v1.1.7] a link to diff page between the last release:
```
[v1.1.7]: https://github.com/ToranSharma/Duo-Strength/compare/v1.1.6...v1.1.7
```